torch/distributed/fsdp/__init__.py

```diff
@@ -1,19 +1,19 @@
-from .flat_param import FlatParameter
+from .flat_param import FlatParameter as FlatParameter
 from .fully_sharded_data_parallel import (
-    BackwardPrefetch,
-    CPUOffload,
-    FullOptimStateDictConfig,
-    FullStateDictConfig,
-    FullyShardedDataParallel,
-    LocalOptimStateDictConfig,
-    LocalStateDictConfig,
-    MixedPrecision,
-    OptimStateDictConfig,
-    OptimStateKeyType,
-    ShardedOptimStateDictConfig,
-    ShardedStateDictConfig,
-    ShardingStrategy,
-    StateDictConfig,
-    StateDictSettings,
-    StateDictType,
+    BackwardPrefetch as BackwardPrefetch,
```

Comment:
hmmm wait, didn't expect a ton of changes like this as those seems not the right way to do. Could you describe your problem or do you have a specific repro?

Reply:
Actually can we do `__all__` I think that would also enable public access

Reply:
Done.